tools/pydocgen/pydocgen/__init__.py

```diff
@@ -70,6 +73,23 @@ class Context(NamedTuple):
     mdoutdir: str
     input: Input
 
+
+class CreateMarkdownInput(NamedTuple):
+    """
+    The CreateMarkdownInput is the input to create_markdown_file.
+    """
+    package_name: str
```

Comment:
Should this be named `title` since it can be either the package name or module name?

Reply:
Yes, good point. I'll update it.